<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>604</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="health">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="energy">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>181</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="pixels">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>88</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="health_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Health</string>
    </property>
   </widget>
   <widget class="QLabel" name="energy_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>181</y>
      <width>44</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixels_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pixels</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="inv_tabs">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Main Bag</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>337</width>
        <height>221</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tile Bag</string>
     </attribute>
     <widget class="QTableWidget" name="tile_bag">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>337</width>
        <height>221</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Count</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>88</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>88</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="race_label">
       <property name="text">
        <string>Race</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="race"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="gender_label">
       <property name="text">
        <string>Gender</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
     <item row="3" column="1">
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <widget class="QRadioButton" name="male">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>57</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Male</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="female">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>0</y>
          <width>72</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Female</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>57</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>88</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>88</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>88</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="action_bar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Count</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="desc_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="desc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <zorder>health</zorder>
   <zorder>energy</zorder>
   <zorder>pixels</zorder>
   <zorder>health_label</zorder>
   <zorder>energy_label</zorder>
   <zorder>pixels_label</zorder>
   <zorder>inv_tabs</zorder>
   <zorder>checkBox</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>female</zorder>
   <zorder>formLayoutWidget_2</zorder>
   <zorder>label</zorder>
   <zorder>doubleSpinBox</zorder>
   <zorder>checkBox_3</zorder>
   <zorder>checkBox_4</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>horizontalSlider_2</zorder>
   <zorder>horizontalSlider_3</zorder>
   <zorder>horizontalSlider_4</zorder>
   <zorder>horizontalSlider_5</zorder>
   <zorder>checkBox_5</zorder>
   <zorder>action_bar</zorder>
   <zorder>comboBox</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>lineEdit_5</zorder>
   <zorder>lineEdit_6</zorder>
   <zorder>lineEdit_7</zorder>
   <zorder>lineEdit_8</zorder>
   <zorder>tableWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="statusTip">
    <string>Open a player save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save player changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
